src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -207,11 +216,20 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
     };
     sendEnterpriseTrackEvent(
       enterpriseId,
-      `${EVENT_NAMES.CONTENT_HIGHLIGHTS.STEPPER_CLOSE_STEPPER_INCOMPLETE}`,
+      `${EVENT_NAMES.CONTENT_HIGHLIGHTS.STEPPER_CLOSE_HIGHLIGHT_MODAL}.clicked`,
       trackInfo,
     );
   };
 
+  const cancelCloseModal = () => {
+    closeCloseAlert();
+    const trackInfo = {};
+    sendEnterpriseTrackEvent(
+      enterpriseId,
+      `${EVENT_NAMES.CONTENT_HIGHLIGHTS.STEPPER_CLOSE_HIGHLIGHT_MODAL}.cancel.clicked`,
+      trackInfo,
```

Comment:
nit: similar comment here; if no event properties to send, no need to define/pass `trackInfo`